src/OmniSharp.Abstractions/Options/FileOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OmniSharp.Options
+{
+    public class FileOptions
+    {
+        public string[] ExcludeSearchPatterns { get; set; } = new[] { "**/node_modules/**/*", "**/bin/**/*", "**/obj/**/*" };
```

Comment:
Should we also add search inclusions?  (in the future perhaps?

Kinda feels like it would be sort of similar to how we used to be able to do things with project.json.

Reply:
What about `"**/.git/**/*"`?

Reply:
that's a good one 👍